source/air/src/printer.rs

```diff
@@ -139,6 +154,21 @@ impl Printer {
                     BinaryOp::Gt => ">",
                     BinaryOp::EuclideanDiv => "div",
                     BinaryOp::EuclideanMod => "mod",
+
+                    BinaryOp::BitXor => "bvxor",
+                    BinaryOp::BitAnd => "bvand",
+                    BinaryOp::BitOr => "bvor",
+                    BinaryOp::BitAdd => "bvadd",
+                    BinaryOp::BitSub => "bvsub",
+                    BinaryOp::BitMul => "bvmul",
+                    BinaryOp::BitDiv => "bvudiv", // unsigned
```

Comment:
In case we forget, we should to make sure these operations in the original context have unsigned operands.

Reply:
I learned that Rust blocks mixed types(signed and unsigned) for %, /, <, >. One strange thing is that Rust allows having mixed types for shifting. In case that both operands are signed integers, % and / are blocked in 'main' branch.
I will make a commit that forces bitvector reasoning to only have unsigned integers on a separate branch.